app/javascript/controllers/application.js

```diff
@@ -1,6 +1,8 @@
 import { Application } from "@hotwired/stimulus"
+import TypedController from "../controllers/typed_controller"
```

Comment:
You have this in the index.js already, you don't need it twice so you can remove iot from here

Reply:
it's not working without this too